packages/mui-utils/package.json

```diff
@@ -29,8 +29,8 @@
     "build": "pnpm build:legacy && pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
     "build:legacy": "node ../../scripts/build.mjs legacy",
     "build:modern": "node ../../scripts/build.mjs modern",
-    "build:node": "node ../../scripts/build.mjs node",
-    "build:stable": "node ../../scripts/build.mjs stable",
+    "build:node": "node ../../scripts/build.mjs node --ignoreTopLevel",
+    "build:stable": "node ../../scripts/build.mjs stable --ignoreTopLevel",
```

Comment:
Why not default to using the same bundling approach as in @mui/material? I thought @mui/icons-material was the exception we had, it's not a great as the alternative but leads to a smaller package size.

Reply:
Okay, looking at it again I think I misunderstood. Will create another PR to fix it.